website/docs/r/alert_channel_webhook.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_webhook"
+description: |-
+  Create and manage Webhook Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_webhook
+
+You can use this resource to create a custom webhook that receives Lacework alert notifications from a Lacework alert channel and forwards those alerts to a third-party application.
+
+-> **Note:**  If the third-party application receiving the HTTP POST request requires an API token, enter the API token as part of the URL eg. https://webhook.com?api-token=123
+
+To find more information about the payload view the lacework support documentation
```

Comment:
Lacework needs to be written capitalized.

Reply:
Ah, missed that. Thanks!